mpsl/doc/fem.rst

```diff
@@ -1,13 +1,13 @@
 .. _mpsl_fem:
 
```

Comment:
Doc changes need to reverted. Those were applied by https://github.com/nrfconnect/sdk-nrfxlib/commit/ae04cb1410bb9e66a3b5c77c3c86e5a0afabdf5e